### DISCUSSION TITLE
How to pass parameter --security-revert=CVE-2023-46809 to babel-node cli

### DISCUSSION THREAD

**Discussion by SSANSH** [babel/babel#16339] [Q&A]
With node we can use parameter  --security-revert=CVE-2023-46809 to revert security fix
But with babel-node i don't find the way to do this.
TypeError: RSA_PKCS1_PADDING is no longer supported for private decryption, this can be reverted with --security-revert=CVE-2023-46809
Note this parameter cannot be pass with NODE_OPTIONS

**Comment by JLHwung** [comment]
Babel-node currently does not support passing through the `--security-revert` option. Currently it was treated as a user app option appended to the `/path/of/your/file.js`.

There is an upstream feature request that it should be supported in the `NODE_OPTIONS`: https://github.com/nodejs/node/issues/52017.

We will wait for Node.js' decisions, since passing through `--security-revert` is a breaking change for babel-node.
